RecoBTag/ONNXRuntime/plugins/ParticleTransformerAK4ONNXJetTagsProducer.cc

```diff
@@ -185,110 +170,63 @@ void ParticleTransformerAK4ONNXJetTagsProducer::make_inputs(btagbtvdeep::Particl
   unsigned offset = 0;
 
   // c_pf candidates
-  auto max_c_pf_n = std::min(features.c_pf_features.size(), (std::size_t)n_cpf_);
+  const auto max_c_pf_n = std::min(features.c_pf_features.size(), (std::size_t)n_cpf_);
```

Comment:
is there a specific reason to make these const?

Reply:
these variables are just used once in the code and they are not passed into any function, so they don't have any risk of being modified... in my understanding it does not matter of I add const or not. If we in general do not make a variable constant if it is not necessary, I will remove it.

Reply:
It's more that unnecessary changes to existing lines of code are discouraged.